src/components/BCHTransaction.vue

```diff
@@ -102,11 +158,15 @@
     >
       <tbody>
         <tr>
-          <td>Index</td>
+          <td>Output index</td>
           <td>{{item.getIndex()}}</td>
         </tr>
+        <tr v-if="item.token">
+          <td>SLP Token</td>
+          <td>{{item.token.isMintBaton ? "MINT BATON" : item.token.amount}} {{item.token.isMintBaton ? "" : transactionData.token_metadata.ticker}}</td>
+        </tr>
         <tr>
-          <td>Value</td>
+          <td>BCH Satoshis</td>
```

Comment:
I like changing 'Value' to 'Satoshis'. Don't think we need BCH.

Reply:
done in 5bec282.